### PR TITLE
Feature/allow backup cronjobs to be run by a k8s cronjob ams cp

### PR DESCRIPTION
**Pull request by LorenzoJokhan**
# Description

This PR adds a new entry point (backup.js) to run backups from. This allows us to create a kubernetes cronjob (see https://github.com/openstad/openstad-kubernetes/pull/101) to perform the backups outside of the API pod.

## Type of change

New feature

## Tests

Tested locally

### CODE REVIEW

**Review by nlsvgtr — APPROVED** [review]
Note: I think this should be removed from the API and maybe setup as a independent service. Will add that to the Headless todo list.

**Comment by rudivanhierden** [comment]
> Note: I think this should be removed from the API and maybe setup as a independent service. Will add that to the Headless todo list.

Agreed! But for now this is a workable solution.

File: src/cron/mongodb_s3_backups.js [hunk 1]
```diff
@@ -1,76 +1,60 @@
...
 
-            s3.putObject(params, function(err, data) {
-                if (err) console.log(err, err.stack);
-                else     console.log(data);
-            });
-          }
-      });
-    }
+      try {
+        await s3.uploadFile(tempFile, fileNameInS3);
```
Comment (Badmuts) [inline src/cron/mongodb_s3_backups.js:48]: Can we also define how long this object is stored for? For example expire after 30 days?

Comment (rudivanhierden) [reply src/cron/mongodb_s3_backups.js:48]: Unfortunately not, we can define how long the object is cached for, but that will not delete the object after expiration. You can implement this through configuration of the S3 environment (for AWS: https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lifecycle-mgmt.html).

If we want to move this responsibility to the API (which is understandable), I think we're stuck with a cronjob type approach.

**Pull request merged**